### PR TITLE
Update version of NPM and packages

### PR DESCRIPTION
**Pull request by pspeter3**
This is just to make sure the code is using up to date packages before a conversion to TypeScript.

### CODE REVIEW

**Comment by jbucaran** [comment]
@pspeter3 I think we don't need to include the lock file for the same reasons given here:

- https://github.com/hyperapp/hyperapp/pull/223

**Comment by pspeter3** [comment]
Interesting, I would have made the opposite argument. Since we have no runtime dependencies, we don't need to worry about lock files affecting downstream customers. On top of that, having a lock file should increase determinism and reduce the number of issues where developers are having inconstancy locally. Up to you though.

**Comment by jbucaran** [comment]
If we need it, we can always add it later right?

**Comment by pspeter3** [comment]
Yep, sounds good.

**Comment by pspeter3** [comment]
I've updated with your requests

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,11 @@
-yarn.lock
-example*
-.vscode
-yarn.lock
+# Artifacts
```
Comment (jbucaran) [inline .gitignore]: @pspeter3 No yarn.lock? I may still be using yarn.

File: package.json [hunk 2]
```diff
@@ -18,12 +18,12 @@
   ],
   "scripts": {
     "test": "exit 0",
-    "build": "rollup -cm -n picodom -f umd -i index.js -o dist/picodom.js",
-    "format": "prettier --semi false --write index.js"
+    "build": "rollup --config --format umd --input index.js --name picodom --output dist/picodom.js --sourcemap",
```
Comment (jbucaran) [inline package.json]: @pspeter3 Can you go back to using the short form of the --flags? 🙏

Comment (pspeter3) [inline package.json]: My thought was that the longer form was easier to read. I can just scrap this PR.

**Comment by jbucaran** [comment]
Thanks @pspeter3! 👍

**Pull request merged**